Pallet Name Customisation (#4806)
paritytech/polkadot-sdk: templates/minimal/README.md

* 🛠️ Depending on your operating system and Rust version, there might be additional
packages required to compile this template - please take note of the Rust compiler output.

## Customisation
In case you would like to change the pallet's name from  `pallet-minimal-template` to `pallet-test` for example, you would need to implement the following changes:
- Change the pallet folder name from `template` to `test` for consistency

> - Also for consistency, in `/runtimesrc/lib.rs`, change from `pub type Template` to `pub type Test`, and don't forget to also customize the corresponding comments.

Reviewer: ```suggestion
- Also for consistency, in `/runtime/src/lib.rs`, change from `pub type Template` to `pub type Test`, and don't forget to also customize the corresponding comments.
```

* 🛠️ Depending on your operating system and Rust version, there might be additional
packages required to compile this template - please take note of the Rust compiler output.

## Customization

> In case you would like to change the pallet's name from  `pallet-minimal-template` to `pallet-test`, you would need to implement the following changes:

Reviewer 2: ```suggestion
In case you would like to change the pallet's name from  `pallet-minimal-template` to `pallet-test`, 
you would need to implement the following changes:
```

* Change the pallet folder name from `template` to `test` for consistency

> * Also for consistency, in `/runtime/src/lib.rs`, change from `pub type Template` to `pub type Test`, and don't forget to also customize the corresponding comments.

Reviewer 2: ```suggestion
* Also for consistency, in `/runtime/src/lib.rs`, change from `pub type Template` to `pub type Test`, 
and don't forget to also customize the corresponding comments.
```
